app/views/iterations/show.html.erb

```diff
@@ -18,6 +18,7 @@
       <%= with_default(@iteration.description, text: 'Description not provided', formatter: :simple_format) %>
     </div>
 
+    <% if !@iteration.status_planned? %>
```

Comment:
Is there a reason we've hidden this for planned iterations? Concern being that users can't see dates they've set for a planned iteration unless they are editing it.

Reply:
You're right, but it needs a bit of rejigging so that it doesn't say 'Progress' when you've not actually committed yet.

The logic for these views is getting pretty complex. I wonder whether we need to just make it a simple three-way switch. It wouldn't by as DRY, but it will be easier to read. What do you think?

Reply:
Good point. On complexity I'd suggest exploring extracting components from the view into ruby objects - here's an interesting library used by GitHub and looks due for rails 6.1 https://github.com/github/actionview-component

Might still need a switch or similar in the view but components could be a useful refactoring/learning opp nonetheless.